### ISSUE TITLE
Mail app won't accept address with "." in it

### ISSUE THREAD

**Issue by dom134** [nextcloud/mail#155]
### Steps to reproduce

Try to apply settings for an email address with a "." in it such as [john.smith@isp.com](url) 
### Expected behaviour

After apply settings emails should be received
### Actual behaviour

Message shows: 'Error while loading the selected account'. Other Nextcloud users with other email addresses, but same ISP connect
I thought it might be similar to (https://github.com/owncloudarchive/mail/issues/626), but I have other accounts with sub-folders in the inbox that are accessible.
### Mail app

**Mail app version:** (see apps admin page)
- mail: 0.6.0
  but also an issue in the previous release

**Mailserver or service:** (e.g. Outlook, Yahoo, Gmail, Exchange,...)
ntlworld.com [forerunner of virginmedia]

**Transport security - IMAP:** (None, SSL, TLS, STARTTLS) 
ssl/tls
**Transport security - SMTP:** (None, SSL, TLS, STARTTLS)
ssl/tls
**Number of accounts:**
Only affects accounts with a "."

**Mail app version build date:** (only if you are using a Nightly Build)
### Server configuration

**Operating system**:
Ubuntu 16.04
**Web server:**
Apache 2
**Database:**
MySQL
**PHP version:**
7.0
**Version:** (see admin page)
10.0.1
**Updated from an older version or fresh install:**
Updated from 9 and 10.0
**List of activated apps:**
Enabled:
- activity: 2.3.2
- admin_audit: 1.0.0
- audioplayer: 1.2.2
- calendar: 1.4.0
- comments: 1.0.0
- contacts: 1.4.0.0
- dav: 1.0.1
- federatedfilesharing: 1.0.1
- federation: 1.0.1
- files: 1.5.2
- files_pdfviewer: 0.8.1
- files_sharing: 1.0.0
- files_texteditor: 2.1
- files_trashbin: 1.0.0
- files_versions: 1.3.0
- files_videoplayer: 0.9.8
- firstrunwizard: 1.1
- gallery: 15.0.0
- mail: 0.6.0
- music: 0.3.12
- notifications: 0.3.0
- password_policy: 1.0.0
- provisioning_api: 1.0.0
- richdocuments: 1.1.7
- serverinfo: 1.1.1
- survey_client: 0.1.5
- systemtags: 1.0.2
- tasks: 0.9.3
- templateeditor: 0.1
- theming: 1.0.1
- updatenotification: 1.0.1
- workflowengine: 1.0.1
  Disabled:
- documents
- encryption
- external
- files_accesscontrol
- files_automatedtagging
- files_external
- files_retention
- user_external
- user_ldap
- user_saml

```
**The content of config/config.php:**
<?php
$CONFIG = array (
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'updatechecker' => false,
  'instanceid' => 'instance',
  'passwordsalt' => 'Password',
  'secret' => 'Secret',
  'updater.secret' => 'Secret',
  'trusted_domains' => 
  array (
    0 => '192.168.0.1',
    1 => 'www.website.com',
  ),
  'datadirectory' => '/var/www/nextcloud/data',
  'overwrite.cli.url' => 'http://192.168.0.1/nextcloud',
  'dbtype' => 'mysql',
  'version' => '9.1.1.5',
  'dbname' => 'ocdb',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'oc_admin',
  'dbpassword' => 'password',
  'loglevel' => '2',
  'logfile' => '/var/www/nextcloud/data/nextcloud.log',
  'logtimezone' => 'UTC',
  'log_authfailip' => true,
  'installed' => true,
  'mail_smtpmode' => 'smtp',
  'mail_from_address' => 'first.last',
  'mail_domain' => 'ntlworld.com',
  'mail_smtphost' => 'smtp.ntlworld.com',
  'mail_smtpport' => '25',
  'maintenance' => false,
  'theme' => '',
  'appstore.experimental.enabled' => true,
);


#### Client configuration
**Browser:**
Mozilla and Chrome
**Operating system:**
Ubuntu 16.04 and Windows 7

#### Logs
##### Web server error log
```

Insert your webserver log here

```

##### Server log (data/nextcloud.log)
{"reqId":"iDZrSaDRpUV3SUu\/KRar","remoteAddr":"88.88.888.888","app":"PHP","message":"include(): Failed opening 'richdocuments\/personal.php' for inclusion (include_path='\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Translation:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Util:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Nls:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Exception:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Date:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Stream_Filter:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Stream:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Socket_Client:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Stream_Wrapper:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Support:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Crypt_Blowfish:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Secret:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Text_Flowed:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Mime:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Idna:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Mail:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_ListHeaders:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Imap_Client:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Smtp:\/var\/www\/nextcloud\/apps\/mail\/vendor\/pear-pear.horde.org\/Horde_Text_Filter:\/var\/www\/nextcloud\/3rdparty\/pear\/console_getopt:\/var\/www\/nextcloud\/3rdparty\/pear\/pear_exception:\/var\/www\/nextcloud\/3rdparty\/pear\/pear-core-minimal\/src:\/var\/www\/nextcloud\/3rdparty\/pear\/archive_tar:\/var\/www\/nextcloud\/3rdparty\/phpseclib\/phpseclib\/phpseclib:\/var\/www\/nextcloud\/lib\/private:\/var\/www\/nextcloud\/config\/:\/var\/www\/nextcloud\/3rdparty:\/var\/www\/nextcloud\/apps:.:\/usr\/share\/php:\/var\/www\/nextcloud') at \/var\/www\/nextcloud\/lib\/private\/legacy\/app.php#759","level":3,"time":"2016-10-15T16:40:01+00:00","method":"GET","url":"\/nextcloud\/index.php\/settings\/personal","user":"admin"}

{"reqId":"Or\/jmgzSX+P4gzxTQspG","remoteAddr":"88.88.888.888","app":"index","message":"Exception: {\"Exception\":\"Horde_Imap_Client_Exception_ServerResponse\",\"Message\":\"IMAP error reported by server.\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/vendor\\\/pear-pear.horde.org\\\/Horde_Imap_Client\\\/Horde\\\/Imap\\\/Client\\\/Socket.php(4302): Horde_Imap_Client_Socket->_getLine(Object(Horde_Imap_Client_Interaction_Pipeline))\\n#1 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/vendor\\\/pear-pear.horde.org\\\/Horde_Imap_Client\\\/Horde\\\/Imap\\\/Client\\\/Socket.php(4230): Horde_Imap_Client_Socket->_sendCmdChunk(Object(Horde_Imap_Client_Interaction_Pipeline), Array)\\n#2 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/vendor\\\/pear-pear.horde.org\\\/Horde_Imap_Client\\\/Horde\\\/Imap\\\/Client\\\/Socket.php(1749): Horde_Imap_Client_Socket->_sendCmd(Object(Horde_Imap_Client_Interaction_Pipeline))\\n#3 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/vendor\\\/pear-pear.horde.org\\\/Horde_Imap_Client\\\/Horde\\\/Imap\\\/Client\\\/Base.php(1726): Horde_Imap_Client_Socket->_status(Array, 1)\\n#4 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/mailbox.php(223): Horde_Imap_Client_Base->status(Array, 1)\\n#5 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/mailbox.php(230): OCA\\\\Mail\\\\Mailbox->getStatus(1)\\n#6 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/account.php(544): OCA\\\\Mail\\\\Mailbox->getTotalMessages()\\n#7 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/account.php(575): OCA\\\\Mail\\\\Account->guessBestMailBox(Array)\\n#8 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/account.php(413): OCA\\\\Mail\\\\Account->getSpecialFolder('archive', true)\\n#9 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/account.php(609): OCA\\\\Mail\\\\Account->getSpecialFoldersIds(false)\\n#10 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/account.php(337): OCA\\\\Mail\\\\Account->localizeSpecialMailboxes()\\n#11 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/account.php(349): OCA\\\\Mail\\\\Account->getMailboxes()\\n#12 \\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/lib\\\/controller\\\/folderscontroller.php(62): OCA\\\\Mail\\\\Account->getListArray()\\n#13 [internal function]: OCA\\\\Mail\\\\Controller\\\\FoldersController->index(4)\\n#14 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(160): call_user_func_array(Array, Array)\\n#15 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/Http\\\/Dispatcher.php(90): OC\\\\AppFramework\\\\Http\\\\Dispatcher->executeController(Object(OCA\\\\Mail\\\\Controller\\\\FoldersController), 'index')\\n#16 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/App.php(111): OC\\\\AppFramework\\\\Http\\\\Dispatcher->dispatch(Object(OCA\\\\Mail\\\\Controller\\\\FoldersController), 'index')\\n#17 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/AppFramework\\\/Routing\\\/RouteActionHandler.php(47): OC\\\\AppFramework\\\\App::main('OCA\\\\\\\\Mail\\\\\\\\Contro...', 'index', Object(OC\\\\AppFramework\\\\DependencyInjection\\\\DIContainer), Array)\\n#18 [internal function]: OC\\\\AppFramework\\\\Routing\\\\RouteActionHandler->__invoke(Array)\\n#19 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/private\\\/Route\\\/Router.php(293): call_user_func(Object(OC\\\\AppFramework\\\\Routing\\\\RouteActionHandler), Array)\\n#20 \\\/var\\\/www\\\/nextcloud\\\/lib\\\/base.php(983): OC\\\\Route\\\\Router->match('\\\/apps\\\/mail\\\/acco...')\\n#21 \\\/var\\\/www\\\/nextcloud\\\/index.php(48): OC::handleRequest()\\n#22 {main}\",\"File\":\"\\\/var\\\/www\\\/nextcloud\\\/apps\\\/mail\\\/vendor\\\/pear-pear.horde.org\\\/Horde_Imap_Client\\\/Horde\\\/Imap\\\/Client\\\/Socket.php\",\"Line\":4590}","level":3,"time":"2016-10-15T16:41:07+00:00","method":"GET","url":"\/nextcloud\/index.php\/apps\/mail\/accounts\/4\/folders","user":"admin"}

{"reqId":"lF607357gk\/BM3wu5cA+","remoteAddr":"88.88.888.888","app":"mail","message":"Creating account failed: Error connecting to mail server.","level":3,"time":"2016-10-15T16:45:37+00:00","method":"POST","url":"\/nextcloud\/index.php\/apps\/mail\/accounts","user":"admin"}
{"reqId":"T7EA3kQVmLMqFu2Yyl1X","remoteAddr":"88.88.888.888","app":"mail","message":"Creating account failed: Error connecting to mail server.","level":3,"time":"2016-10-15T16:45:57+00:00","method":"POST","url":"\/nextcloud\/index.php\/apps\/mail\/accounts","user":"admin"}
{"reqId":"5E75n5RP3GO+eq\/KMROQ","remoteAddr":"88.88.888.888","app":"mail","message":"Creating account failed: Error connecting to mail server.","level":3,"time":"2016-10-15T16:46:34+00:00","method":"POST","url":"\/nextcloud\/index.php\/apps\/mail\/accounts","user":"admin"}

##### Horde IMAP log (data/horde_imap.log)
------------------------------
>> Sat, 15 Oct 2016 17:06:02 +0000
>> Connection to: imap://imap.ntlworld.com:993/
>> Server connection took 0.1291 seconds.
S: * OK Virgin Media IMAP4 server ready [ e4c558782NTL ].
C: 1 CAPABILITY
S: * CAPABILITY IMAP4rev1 NAMESPACE
S: 1 OK CAPABILITY completed.
>> Command 1 took 0.0677 seconds.
C: 2 LOGIN dom.walden@ntlworld.com [PASSWORD]
S: 2 OK [CAPABILITY IMAP4rev1 LITERAL+ SASL-IR LOGIN-REFERRALS ID ENABLE IDLE SORT SORT=DISPLAY THREAD=REFERENCES THREAD=REFS THREAD=ORDEREDSUBJECT MULTIAPPEND URL-PARTIAL CATENATE UNSELECT CHILDREN NAMESPACE UIDPLUS LIST-EXTENDED I18NLEVEL=1 CONDSTORE QRESYNC ESEARCH ESORT SEARCHRES WITHIN CONTEXT=SEARCH LIST-STATUS BINARY MOVE SPECIAL-USE QUOTA] Logged in
>> Command 2 took 0.1995 seconds.
>> CACHE: Using the OCA\Mail\Cache\Cache storage driver.
C: 3 ENABLE QRESYNC
C: 4 LOGOUT
S: * ENABLED QRESYNC
S: 3 OK Enabled (0.000 + 0.000 secs).
>> Command 3 took 0.0407 seconds.
S: * BYE Logging out
S: 4 OK Logout completed.
>> Command 4 took 0.0574 seconds.
------------------------------
>> Sat, 15 Oct 2016 17:06:03 +0000
>> Connection to: imap://imap.ntlworld.com:993/
>> Server connection took 0.1172 seconds.
S: * OK Virgin Media IMAP4 server ready [ e4c558782NTL ].
C: 1 CAPABILITY
S: * CAPABILITY IMAP4rev1 NAMESPACE
S: 1 OK CAPABILITY completed.
>> Command 1 took 0.021 seconds.
C: 2 LOGIN dom.walden@ntlworld.com [PASSWORD]
S: 2 OK [CAPABILITY IMAP4rev1 LITERAL+ SASL-IR LOGIN-REFERRALS ID ENABLE IDLE SORT SORT=DISPLAY THREAD=REFERENCES THREAD=REFS THREAD=ORDEREDSUBJECT MULTIAPPEND URL-PARTIAL CATENATE UNSELECT CHILDREN NAMESPACE UIDPLUS LIST-EXTENDED I18NLEVEL=1 CONDSTORE QRESYNC ESEARCH ESORT SEARCHRES WITHIN CONTEXT=SEARCH LIST-STATUS BINARY MOVE SPECIAL-USE QUOTA] Logged in
>> Command 2 took 0.2389 seconds.
>> CACHE: Using the OCA\Mail\Cache\Cache storage driver.
C: 3 ENABLE QRESYNC
C: 4 LIST () "" (*) RETURN (SPECIAL-USE)
S: * ENABLED QRESYNC
S: 3 OK Enabled (0.000 + 0.000 secs).
>> Command 3 took 0.0483 seconds.
S: * LIST (\Noselect) "/" Archives
S: * LIST () "/" Archives/2010
S: * LIST () "/" Archives/2012
S: * LIST () "/" Archives/2013
S: * LIST () "/" Archives/2014
S: * LIST () "/" Archives/2015
S: * LIST () "/" Archives/Trash
S: * LIST (\Drafts) "/" Drafts
S: * LIST () "/" stuff
S: * LIST () "/" things
S: * LIST () "/" "Follow up"
S: * LIST () "/" other
S: * LIST () "/" Gardening
S: * LIST () "/" House
S: * LIST () "/" "Junk E-mail"
S: * LIST () "/" Misc
S: * LIST () "/" "friends"
S: * LIST () "/" Receipts
S: * LIST () "/" all
S: * LIST (\Sent) "/" Sent
S: * LIST (\Noselect) "/" [Gmail]
S: * LIST () "/" "[Gmail]/All Mail"
S: * LIST () "/" [Gmail]/Chats
S: * LIST () "/" [Gmail]/Drafts
S: * LIST () "/" [Gmail]/Important
S: * LIST () "/" "[Gmail]/Sent Mail"
S: * LIST () "/" [Gmail]/Starred
S: * LIST (\Noselect) "/" [Imap]
S: * LIST () "/" [Imap]/Archives
S: * LIST () "/" children
S: * LIST () "/" work
S: * LIST (\Trash) "/" Trash
S: * LIST (\Junk) "/" Spam
S: * LIST () "/" INBOX
S: 4 OK List completed (0.000 + 0.000 secs).
>> Command 4 took 0.2066 seconds.
C: 5 STATUS Drafts (MESSAGES)
S: * STATUS Drafts (MESSAGES 1)
S: 5 OK Status completed (0.000 + 0.000 secs).
>> Command 5 took 0.0448 seconds.
C: 6 STATUS Drafts (MESSAGES)
S: * STATUS Drafts (MESSAGES 1)
S: 6 OK Status completed (0.000 + 0.000 secs).
>> Command 6 took 0.0453 seconds.
C: 7 STATUS [Gmail]/Drafts (MESSAGES)
S: * STATUS [Gmail]/Drafts (MESSAGES 0)
S: 7 OK Status completed (0.000 + 0.000 secs).
>> Command 7 took 0.0912 seconds.
C: 8 STATUS Archives/Trash (MESSAGES)
S: * STATUS Archives/Trash (MESSAGES 3995)
S: 8 OK Status completed (0.000 + 0.000 secs).
>> Command 8 took 0.0468 seconds.
C: 9 STATUS Archives/Trash (MESSAGES)
S: * STATUS Archives/Trash (MESSAGES 3995)
S: 9 OK Status completed (0.000 + 0.000 secs).
>> Command 9 took 0.0419 seconds.
C: 10 STATUS Trash (MESSAGES)
S: * STATUS Trash (MESSAGES 220)
S: 10 OK Status completed (0.000 + 0.000 secs).
>> Command 10 took 0.0448 seconds.
C: 11 STATUS [Imap]/Archives (MESSAGES)
S: * STATUS [Imap]/Archives (MESSAGES 0)
S: 11 OK Status completed (0.000 + 0.000 secs).
>> Command 11 took 0.0677 seconds.
C: 12 STATUS [Imap]/Archives (MESSAGES)
S: * STATUS [Imap]/Archives (MESSAGES 0)
S: 12 OK Status completed (0.000 + 0.000 secs).
>> Command 12 took 0.042 seconds.
C: 13 STATUS Archives (MESSAGES)
S: 13 NO Mailbox doesn't exist: Archives (0.000 + 0.000 secs).
>> Command 13 took 0.044 seconds.
C: 14 LOGOUT
S: * BYE Logging out
S: 14 OK Logout completed.
>> Command 14 took 0.0623 seconds.
</details>

##### Horde SMTP log (data/horde_smtp.log)
------------------------------
>> Sat, 15 Oct 2016 17:06:02 +0000
>> Connection to: smtp://smtp.ntlworld.com:465
S: 220 know-smtprelay-9-imp bizsmtp ESMTP server ready
C: EHLO derg
S: 250-know-smtprelay-9-imp hello [88.88.888.888], pleased to meet you
S: 250-HELP
S: 250-AUTH LOGIN PLAIN
S: 250-SIZE 52000000
S: 250-8BITMIME
S: 250 OK
C: AUTH LOGIN
S: 334 VXNlcm5hbWU6
C: ZG9tLndhbGRlbkBudGx3b3JsZC5jb20=
S: 334 UGFzc3dvcmQ6
[AUTH Command - method: LOGIN; username: first.@ntlworld.com]
S: 235 ... authentication succeeded
C: QUIT
S: 221 know-smtprelay-9-imp bizsmtp closing connection


##### Browser log
<details>
```

Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...

```
</details>
```

**Comment by ChristophWurst** [comment]
> Other Nextcloud users with other email addresses, but same ISP connect

Did they use the very same settings for IMAP and SMTP?

I have multiple accounts with a dot in it and have never had that problem.

**Comment by dom134** [comment]
The manual configuration does not seem to work for my accounts, but the automatic login works fine and picks up my other accounts.
The dot in the name is just my hypothesis: I might be barking up the wrong tree!

**Comment by ChristophWurst** [comment]
Strange. Do you have access to the database? If yes, please compare the manually configured account with the automatically configured one (`oc_mail_accounts` table, assuming `oc_` is your table prefix). Would be interesting to see where that differs

**Comment by dom134** [comment]
Hello Christoph,
My other ntlworld account shows this in mysql:

> |  6 | name | name    | all_one_word@ntlworld.com   | imap.ntlworld.com    | 993          | ssl              | all_one_word@ntlworld.com   | gobbledegook|gobbledegook|gobbledegook | smtp.ntlworld.com    | 465           | ssl               | all_one_word@ntlworld.com   | gobbledegook|gobbledegook|gobbledegook |

the first.last@ntlworld.com account is not shown in the database

**Comment by ChristophWurst** [comment]
> the first.last@ntlworld.com account is not shown in the database

From your description I assumed that you can add the account manually, but then you cannot use the app because it fails loading the account. In that case the account info of the newly added account should be in the database too.

Did you use the same ports + ssl modes on the manual setup?

**Comment by dom134** [comment]
Yes, I used the same ports on the manual setup.  Neither account could be created manually, but the all_one_word@ntlworld.com account was set up using the automatic setup.
Creating the first.last@ntlworld.com account manually just didn't do anything, but when I switched to automatic, that's when I get the 'Error while loading the selected account'.  With 0.6 mail version the account name is shown on the left-hand panel, but the righthand side just has a thinks-circle.  I just delete the account then.

**Comment by dom134** [comment]
Apologies, the notification I get is:

> Error while creating the account: Error

**Comment by ChristophWurst** [comment]
> C: 13 STATUS Archives (MESSAGES)
> S: 13 NO Mailbox doesn't exist: Archives (0.000 + 0.000 secs).

This looks fishy. At the beginning of the list I see the folder mailbox listing and

> S: * LIST (\Noselect) "/" Archives

which means the app tries to get the status of a mailbox that has the `\Noselect` flag set. That could possibly cause the failure… :thinking:

**Comment by ChristophWurst** [comment]
This should be fixed by #271.
